src/devices/Pca9685/Bit.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Pca9685
+{
+    internal enum Bit : byte
```

Comment:
please add some comments😀

Reply:
this needs different name describing what this means - `Bit` is a bit generic :smile: